dashboard/src/main/webapp/vue.config.js

```diff
@@ -16,4 +16,16 @@
  */
 
 module.exports ={
+    // 可以通过配置 vue.config.js 文件来设置代理，以将请求代理到后端服务器。
```

Comment:
Could you translate this sentence into english?

Reply:
> Could you translate this sentence into english?

done.